rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMImagesUnbundled.java

```diff
@@ -2,7 +2,9 @@
 
 public class RMImagesUnbundled {
 
-    public static final String LOGO_32 = "logo_32.png";
     public static final String LOGO_350 = "logo_350.png";
+    public static final String AE_LOGO = "AE-Logo.png";
+    public static final String AE_ICON = "AE-Icon.png";
```

Comment:
Can we rename to CLIENT_LOGO ? or something that makes it easier to understand that is the logo that can change?